### PR TITLE
added anuvar.R and test file with comments 

### PR DESCRIPTION
**Pull request by Pruthv1**

### CODE REVIEW

File: tests/testthat/testAnuvar.R [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+NL <- .PlatformLineBreak()
+# ==== END SETUP AND PREPARE ===================================================
+
+
+
+test_that("parameter errors are correctly handled", {
+    # no input
+    expect_error(anuvar())
+    # numeric input
+    expect_error(anuvar(numeric(), testgeneData))
```
Comment (RicardoRam) [inline tests/testthat/testAnuvar.R]: where is testgeneData defined?

Comment (RicardoRam) [inline tests/testthat/testAnuvar.R]: Are testgeneData and testSNV just placeholders?

File: R/anuvar.R [hunk 2]
```diff
@@ -0,0 +1,106 @@
+# importSNV.TCGA.R
+
+#' Import MAF files from source and converts them to rSNV files.
```
Comment (RicardoRam) [inline R/anuvar.R]: Function documentation looks good

Comment (hyginn) [inline R/anuvar.R:3]: ???

File: R/anuvar.R [hunk 3]
```diff
@@ -0,0 +1,106 @@
+# importSNV.TCGA.R
```
Comment (hyginn) [inline R/anuvar.R:1]: ???

File: R/anuvar.R [hunk 4]
```diff
@@ -0,0 +1,106 @@
...
+
+    # Validate that SNV table contatins valid column names
+    # Validate GeneData table contains valid column names
+
+    # ==== READ DATA ===========================================================
+
+    # 1. FIND rows in SNV mutation table that exists in GeneData table
+
+    # 2. ADD extra information from the GeneData table to SNV table
+    # # exonStart - chromosomal coordinates of all exon first nucleotides for the canonical cDNA sequence
```
Comment (hyginn) [inline R/anuvar.R:68]: What would that be used for?

File: R/anuvar.R [hunk 5]
```diff
@@ -0,0 +1,106 @@
...
+    # # exonEnd - chromosomal coordinates of all exon last nucleotides for the canonical cDNA sequence
+    # # map - function to map a chromosome coordinate to a cDNA position for this gene
```
Comment (hyginn) [inline R/anuvar.R:70]: What would that be used for?

File: R/anuvar.R [hunk 6]
```diff
@@ -0,0 +1,106 @@
...
+    # # cDNA - canonical cDNA sequence of the gene
+    # # phastCons - phastCons conservation scores (* 1000) for each cDNA nucleotide (from UCSC)
+    # # phyloP - phyloP conservation scores (* 1000) for each cDNA nucleotide (from UCSC)
+    # # 3D map - rownames are the PDB-ID/chain, columns are the first and last residue for which 3D coordinates have been mapped.
+    # # 3D - list indices are PDB-ID/chain. For each PDB-ID/chain there is a data frame in which rownames a
+    # #      are the gene's sequence positions for which 3D coordinates have been mapped to this structure.
+    # #      Columns are the the structure residue ID, amino acid type, and the x, y, and z coordinates of
+    # #      the Cα atom and of the side-chain centroid.
+
```
Comment (hyginn) [inline R/anuvar.R:79]: Did you have a look at the MutSig CV paper? What annotations should anuvar actually provide?

File: R/anuvar.R [hunk 7]
```diff
@@ -0,0 +1,106 @@
...
+#'   enabled. TRUE by default.
+#'
+#' @family
+#'
+#'   ## @seealso \code{\link{anuvar}} anuvar()
+#'
+#'   ## @examples ## \dontrun {
+#'   ## anuvar(SNV, geneData) ## }
+#' @export
+#' @return SNV table with annotations added from the GeneData table.
```
Comment (hyginn) [inline R/anuvar.R:28]: A separate table?

**Comment by hyginn** [comment]
Reviewed

**Pull request closed without merging**